src/models/CommonModel.ts

```diff
@@ -281,9 +281,9 @@ export class CommonModel extends CommonSchema<CommonModel> {
         for (const [pattern, patternModel] of Object.entries(mergeFromPatternProperties)) {
           if (mergeToPatternProperties[`${pattern}`] !== undefined) {
```

Comment:
```suggestion
          if (mergeToPatternProperties[String(pattern)] !== undefined) {
```

Could potentially introduce a variable here since it is used 2 places, but up to you 🙂

Reply:
aah! missed it😅